bin/deseq2.R

```diff
@@ -4,16 +4,15 @@ library("gplots")
 library("ggplot2")
 library("ReportingTools")
 library("pheatmap")
-library("biomaRt")
 library("svglite")
-library("piano")
 library("apeglm")
 library("EnhancedVolcano")
 library("regionReport")
 library("stringr")
-library("WebGestaltR")
 library("snowfall")
 library("openxlsx")
+library("foreach")
```

Comment:
we also need to update the Docker container then... or? Did you test this @fischer-hub